### ISSUE TITLE
Options for diocesan propers

### ISSUE THREAD

**Issue by peterdiekertz** [DivinumOfficium/divinum-officium#423]
I must admit I do not know how to achieve this technically, but it would probably be a good idea to somehow make it possible to observe local feasts (apart from the dedication of the church, which is already an option).
Maybe there could be a new option for the calendar, with the standard being the General Calendar and the possibility to select those diocesan calendars which are known to us and added into the code.

**Comment by gregordick** [comment]
This will be possible in principle once the `calcalc` branch has been merged.  It allows support for multiple calendars that inherit from and modify one another.  Most of the work would then be to create an appropriate user interface.

**Comment by peterdiekertz** [comment]
I see.
Does the calcalc branch need help with coding or so (provided one has adequate Perl knowledge)?

**Comment by gregordick** [comment]
Help would certainly be welcome!  Not all of the work requires coding, either: for example, one significant outstanding task is to convert all of the vernacular files to use the new scheme whereby commemorated offices are stored in separate files.

**Comment by peterdiekertz** [comment]
Ok, so this concerns all the files which now contain Saint names in the filename, like <https://github.com/DivinumOfficium/divinum-officium/blob/calcalc/web/www/horas/Latin/Sancti/01-18-Prisca.txt>?

In this case, does it suffice to move St. Prisca's texts from e.g. English/Sancti/01-18 to a new English/Sancti/01-18-Prisca?
Also, are there any problems with the files in calcalc not being updated after changes were done in the main branch?

**Comment by gregordick** [comment]
> Ok, so this concerns all the files which now contain Saint names in the filename, like https://github.com/DivinumOfficium/divinum-officium/blob/calcalc/web/www/horas/Latin/Sancti/01-18-Prisca.txt?

That is correct.

> In this case, does it suffice to move St. Prisca's texts from e.g. English/Sancti/01-18 to a new English/Sancti/01-18-Prisca?

Yes.

> Also, are there any problems with the files in calcalc not being updated after changes were done in the main branch?

From time to time I merge changes from `master` to `calcalc`.  Where the changes affect files that have been split on the latter branch, then there is a bit of manual work involved in the merge (but it doesn't actually happen that often, and it's usually straightforward).

**Comment by peterdiekertz** [comment]
I see. I will try to help with this moving of texts then.

**Comment by luc--** [comment]
Hello, as I’m actually uploading the French translation of the abbaye du Barroux, what should I do with the proper of Saints : do I have to create the files for the proper of France too, even if the functionality is not yet implemented ?

**Comment by gregordick** [comment]
Yes, please do.  I would suggest creating a hierarchy of subdirectories inside `Sancti` for country (and diocese where applicable), in the nomimative and genitive respectively, and including the name of the saint in the filename e.g. `Sancti/Anglia/Westmonasteriensis/11-17-Hugo.txt`.  I used nominatives for saints' names on the `calcalc` branch; genitives would probably have been better, but let's stick with the former.

**Comment by MatheusdSMoraes** [comment]
Hello, I'm new here. I'd know how to put the Brazilian Calendar, with their saints and their prayers
